### ISSUE TITLE
Downlink transmission failed with result Too EARLY

### ISSUE THREAD

**Issue by zouOrange** [TheThingsNetwork/lorawan-stack#5577]
### Summary

Hello, please ask a question, in Class C mode, TTN(NS) sends data too early(NS displays an error message: Failed to transmit downlink Downlink transmission failed with result Too EARLY), resulting in packet loss problems. Why? Thank you!

### Current Situation

Frame counter width
●32 bit
Desired Rx1 delay②
5  sec
Desired Rx1 data rate offset
0
Desired Rx2 data rate index
8
Desired Rx2 frequency
923.3MHz
Desired maximum duty cycle
100%
Factory preset frequencies
Class C timeout②
300 seconds
Status time periodicity
200
messages
86400 seconds

### Desired Situation

Want to give problem analysis and solutions.

### Code of Conduct

- [X] I agree to follow TTN's [Community Code of Conduct](https://github.com/TheThingsNetwork/Manifesto/blob/master/CodeOfConduct.md).

**Comment by KrishnaIyer** [comment]
@zouOrange: Please add proper debugging information that includes device, gateway settings and event logs.

**Comment by zouOrange** [comment]
You mean I set the device information with an error?

**Comment by KrishnaIyer** [comment]
No, I mean please provide us detailed information on your setup and the event logs from The Things Stack Console or CLI.

**Comment by zouOrange** [comment]
![1](https://user-images.githubusercontent.com/108652935/177132050-575a14bd-d751-45cd-9697-6fcb3ba1a914.PNG)
![2](https://user-images.githubusercontent.com/108652935/177132060-8e7896f9-30d4-49cd-af47-c641f7e5f335.PNG)
![3](https://user-images.githubusercontent.com/108652935/177132065-9d51d122-b649-469c-aa12-e3c54d90e48c.PNG)
![4](https://user-images.githubusercontent.com/108652935/177132067-fc83c1f9-924d-4288-9eef-9afa1d24f838.PNG)
This is my configuration information, please take a look.

**Comment by zouOrange** [comment]
This is where I went wrong
![5png](https://user-images.githubusercontent.com/108652935/177133837-10b0190e-fce8-404b-be0d-586ad0ee573d.png)

**Comment by KrishnaIyer** [comment]
Are you using Class-C absolute scheduling? What is the scheduling request?

The Things Stack can buffer messages to be sent to gateways if the gateway rejects downlinks sent in advance.

So, on the console, you could select the "Enable server-side buffer of downlink messages" option and try again. 

Also, gateway Packet Forwarder logs would be really helpful.

**Comment by KrishnaIyer** [comment]
This issue is stale.
